staging/src/k8s.io/apiserver/pkg/endpoints/handlers/watch.go

```diff
@@ -34,12 +34,17 @@ import (
 	"k8s.io/apimachinery/pkg/watch"
 	"k8s.io/apiserver/pkg/endpoints/handlers/negotiation"
 	"k8s.io/apiserver/pkg/endpoints/metrics"
+	apirequest "k8s.io/apiserver/pkg/endpoints/request"
 	"k8s.io/apiserver/pkg/util/wsstream"
 )
 
 // nothing will ever be sent down this channel
 var neverExitWatch <-chan time.Time = make(chan time.Time)
 
+// nothing will ever be sent down this channel, used when the request
+// context does not have a server shutdown signal associated
+var neverShuttingDownCh <-chan struct{} = make(chan struct{})
```

Comment:
out of curiosity, why is this global?

Reply:
Or even just set nil as the shutdown channel. Nil blocks on select.

Reply:
yeah, i don't thin we need it, i removed it, thanks